app/routes/circulars._archive._index.tsx

```diff
@@ -291,7 +324,160 @@ export default function () {
           </Button>
         </Form>
         {featureCircularsFilterByDate && (
-          <DateSelectorButton startDate={startDate} endDate={endDate} />
+          <>
+            <DateSelectorButton
+              startDate={startDate}
+              endDate={endDate}
+              onClick={() => setShowContent(!showContent)}
+              expanded={showContent}
+            />
+            {showContent && (
+              <DetailsDropdownContent className="margin-top-1">
+                <div className="usa-radio maxw-card-xlg">
+                  <div className="display-flex flex-row">
+                    <div className="display-flex flex-column flex-align-start margin-1">
+                      <Radio
+                        id="radio-alltime"
+                        name="radio-date"
+                        value="undefined"
+                        defaultChecked={true}
+                        label="All Time"
+                        className="usa-search__filter-button"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      ></Radio>
+                      <Radio
+                        id="radio-hour"
+                        name="radio-date"
+                        value="hour"
+                        label="Past hour"
+                        className="usa-search__filter-button"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      ></Radio>
+                      <Radio
+                        id="radio-today"
+                        name="radio-date"
+                        value="today"
+                        label="Today"
+                        className="usa-search__filter-button"
+                        onChange={(e) => {
+                          setFuzzyTime(e.target.value)
+                        }}
+                      ></Radio>
+                    </div>
+                    <div className="display-flex flex-column margin-1">
```

Comment:
You should be able to use the `<Grid>` component to structure this in such a way that you don't need to manually break the radio buttons into rows.

Reply:
@lpsinger I've been experimenting with this, and I've found that enclosing them into a single grid component doesn't result in the 3x3 of radio buttons I was aiming for. However, I tried to do a similar thing to the responsive layout in the react components library (see [here](https://trussworks.github.io/react-uswds/?path=/docs/components-grid--responsive)), which resulted in something like this:
<img width="463" alt="image" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/57914086/2a399494-1cee-45e6-bd2c-bd5d3660f6da">

Reply:
Yes, that's what I'm suggesting:

```jsx
<Grid row>
  <Grid col={4}
    <Radio ... />
  </Grid>
  <Grid col={4}
    <Radio ... />
  </Grid>
  <Grid col={4}
    <Radio ... />
  </Grid>
  <Grid col={4}
    <Radio ... />
  </Grid>
  ...
</Grid>
```